samcli/cli/main.py

```diff
@@ -54,6 +56,7 @@ def print_info(ctx, param, value):
         "version": __version__,
         "system": gather_system_info(),
         "additional_dependencies": gather_additional_dependencies_info(),
+        "available_beta_feature_env_vars": get_all_experimental_env_vars(),
```

Comment:
Do customers know how to toggle these env vars? Is that documented anywhere?

Reply:
Good catch.

We mentioned about `SAM_CLI_BETA_RUST_CARGO_LAMBDA` in the [Rust Builder page](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/building-rust.html#building-rust-prerequisites). But I couldn't find other env vars mentioned in anywhere else.

I believe we should have a doc talking about beta features or about all SAM CLI env vars.